test/kwokctl/kwokctl_workable_test.sh

```diff
@@ -132,6 +132,18 @@ function test_kwok_controller_port() {
     fi
 }
 
+function test_etcd_port() {
+    local result
+    result=$(curl -s http://127.0.0.1:2400/health)
+
+    if [[ "$(echo "${result}" | grep -o '"health"' | wc -c)" = 0 ]]; then
```

Comment:
Locally, the `$result` would be `{"health":"true","reason":""}`, but on CI, the result was:
`{"health: "true"}` (also with space between `:` and `"`.
So I thought just being able to fetch `"health"` was good enough sign that we are able to talk to the etcd

Reply:
I think it's OK